### ISSUE TITLE
Hyperparameter settings for CIFAR-100

### ISSUE THREAD

**Issue by zhangxin-xd** [VILA-Lab/SRe2L#10]
Thank you so much for sharing this excellent work! 

I notice that you have conducted experiments on CIFAR-100. Are the hyperparameter settings the same as Imagenet-1K or tiny-Imagenet? 

Looking forward to your response!

**Comment by zeyuanyin** [comment]
Thank you for your interest in our work. We have provided the implementation details for CIFAR-100 in Section B.1 of the Appendix in our CDA paper (https://arxiv.org/pdf/2311.18838).

**Comment by zhangxin-xd** [comment]
Thanks for the response!  I note that Table 11 reports the results of 800 ep. Does that mean the network should be trained on synthetic data for 800 ep?

**Comment by zeyuanyin** [comment]
Yes, you can train the ResNet-18 model on synthetic data for 800 epochs to reproduce results in Table 11.

**Comment by zhangxin-xd** [comment]
Hi I have trained SRe2L code with the provided hyperparameters, but the acc is only 47.57 for cifar-100 50ipc (which is 49.37 in SRe2L paper). Should I extend the recovery iteration which is set to 1000 now.

**Comment by szq0214** [comment]
Hi @zhangxin-xd Could you please check the configurations for CIFAR-100 detailed in our latest paper, available at https://openreview.net/pdf?id=5Fgdk3hZpb (Table 10). These settings achieved a 51.35% accuracy with an IPC 50.
